TouchPortalAPI/client.py

```diff
@@ -356,11 +371,11 @@ def choiceUpdate(self, choiceId:str, values:list):
         This updates the list of choices in a previously-declared TP State with id `stateId`.
         See TP API reference for details on updating list values.
         """
-        if choiceId:
-            if isinstance(values, list):
-                self.send({"type": "choiceUpdate", "id": choiceId, "value": values})
-            else:
-                raise TypeError(f'choiceUpdate() values argument needs to be a list not a {type(values)}')
+        if isinstance(values, list):
```

Comment:
I think we want/need to keep the check here for `if choiceId:`, especially if that's going to be used as a key in a dict.